README.md

```diff
@@ -1 +1,44 @@
-# NodeJs-tutorial
\ No newline at end of file
+# NodeJs Project
+## An App To Manage Tutorials.
+
+## Features
+APIs:
+- Create a Tutorial
+- Update a Tutorial by ID
+- Delete a Tutorial by ID
+- Get all Tutorials
+- Able to search by Title
+- Able to sort by crated/updated date (Default updated DESC)
+
+## Installation
+TutorialsNodeJs requires [Node.js](https://nodejs.org/) v16+ to run.
+
+```sh
+$ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
+$ nvm install 16.13.1
+$ nvm use 8.1.2
+```
+## Clone this repo (and cd to it)
+
+```sh
+$ git clone https://github.com/JainishaParmar/NodeJs-tutorial.git
+$ cd nodeJsDem0-tutorial
+```
+## Install the dependencies and devDependencies and start the server.
+
+```sh
+npm install
+```
+## Load env vars
+
+```sh
```

Comment:
Here, you should explain to set required environment variables, Make sure to not expose any real secrets you can still use local host URL and all.
```
MONGO_URL="mongodb://localhost:27017/tutorials"
```